core/tools/mavlink_router/bootstrap.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+LOCAL_BINARY_PATH="/usr/bin/mavlink-routerd"
+VERSION="companion-core-devlopment-0.1"
```

Comment:
```suggestion
VERSION="companion-core-development-0.1"
```

Reply:
Ops, I'll need to create the tag and build for that.